src/utils/src/Include_i.h

```diff
@@ -270,6 +270,19 @@ typedef struct {
     logPrintFunc storedLoggerLogPrintFn;
 } FileLogger, *PFileLogger;
 
+typedef struct __ExponentialBackoffRetryStrategyState {
+    ExponentialBackoffRetryStrategyConfig exponentialBackoffRetryStrategyConfig;
+    ExponentialBackoffStatus status;
+    UINT32 currentRetryCount;
+    // The system time at which last retry happened
+    UINT64 lastRetrySystemTime;
+    // The actual wait time for last retry
+    UINT64 lastRetryWaitTime;
+    // Lock to update operations
+    MUTEX retryStrategyLock;
+} ExponentialBackoffRetryStrategyState;
+typedef ExponentialBackoffRetryStrategyState* PExponentialBackoffRetryStrategyState;
```

Comment:
You need a `,` between these two.

Reply:
I see this is differnet, why not just put it on line 283,
```
} ExponentialBackoffRetryStrategyState, *PExponentialBackoffRetryStrategyState
```

like we do for other structs